src/sentry/static/sentry/app/views/settings/components/forms/projectMapperField.tsx

```diff
@@ -46,13 +47,15 @@ export class RenderField extends React.Component<RenderProps, State> {
       value: incomingValues,
       sentryProjects,
       mappedDropdown: {items: mappedDropdownItems, placeholder: mappedValuePlaceholder},
-      nextButton: {url: nextUrl, text: nextButtonText},
+      nextButton: {text: nextButtonText},
       iconType,
       model,
       id: formElementId,
       error,
     } = this.props;
     const existingValues: Array<[number, MappedValue]> = incomingValues || [];
+    const nextUrlOrArray = safeGetQsParam('next');
```

Comment:
you will need to validate the domain on this otherwise someone can potentially use to to do something bad

Reply:
@dcramer good callout, fixed